Ghostscript.NET.PDFA3Converter/PDFA3Converter.cs

```diff
@@ -60,7 +60,7 @@ public class PDFA3Converter
         public PDFA3Converter(String gsdll)
         {
             GSDLLPath = gsdll;
-            RGBICCFilename = "rgb.icc";
+            RGBICCFilePath = System.IO.Path.Combine(Path.GetTempPath(), "rgb.icc");
```

Comment:
can't read embedded icc file, I think that should be `RGBICCFilePath = "rgb.icc"`.